__tests__/component-select.tsx

```diff
@@ -0,0 +1,22 @@
+import Select from 'components/Form/Select'
+import { makeRendererFor } from 'jest.utils'
+
+const selectWithProps = makeRendererFor(Select)
+
+it('renders a Select component', () => {
+  const options = 'abcde'.split('').map(name => ({
+    name,
+    value: name,
+  }))
+
+  const { container } = selectWithProps({
+    disabled: false,
+    options,
+    defaultValue: 'a',
+    value: 'a',
+    onChange: jest.fn(),
+    setTouched: jest.fn(),
+    className: 'test',
+  })
+  expect(container).toMatchSnapshot()
```

Comment:
im confused what this is testing. the values (options) are exactly what is in the snapshot. it's just testing that the snapshot that we ourselves generate has the correct values?